meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -31,8 +32,16 @@ do_install_append() {
 
     install -d ${D}${systemd_unitdir}/system/
     install -m 0644 ${WORKDIR}/ambd.service ${D}${systemd_unitdir}/system
+    
+    install -d ${D}${libdir}/systemd/user/
+    install -m 0644 ${WORKDIR}/ambd_fsa.service ${D}${libdir}/systemd/user/ambd_fsa.service
+    mkdir -p ${D}/home/root/.config/systemd/user/default.target.wants
+    ln -sf ${libdir}/systemd/user/ambd_fsa.service ${D}/home/root/.config/systemd/user/default.target.wants/ambd_fsa.service.service
+
 }
 
-FILES_${PN} += "${systemd_unitdir}/ambd.service"
+FILES_${PN} += "${systemd_unitdir}/system/ambd.service"
+FILES_${PN} += "${libdir}/systemd/user/*.service \
+                /home/root/.config/systemd/user/default.target.wants/*.service"
```

Comment:
No problem.
In fact there is already a service user file /usr/lib/systemd/user/... and his link to /home/....,, I will test without the link. If the service is still started automatically. That is fine for me, or I will see if I can have a workarround to avoid this link. If any case, no, we will see.